config/application.rb

```diff
@@ -63,8 +63,6 @@ class Application < Rails::Application
     # Therefore we disable "per_form_csrf_tokens" for the time being.
     config.action_controller.per_form_csrf_tokens = false
 
-    config.middleware.use SetCookieDomain
```

Comment:
despite being here in the current code, `SetCookieDomain` is only actually used in `production`:

https://github.com/forem/forem/blob/ff244a31eff9d8ad270dd7b0aa4c6ed5eac03223/app/middlewares/set_cookie_domain.rb#L9-L14

As you can see, in any other environment, it's just a pass through call


no need to insert a global middleware when we don't use it, see https://github.com/forem/forem/pull/13793/files#diff-4493b648c41705909d0895e7af5b09ee416e774adf53b042e837975cf08d3a05